gatsby-config.js

```diff
@@ -70,6 +70,16 @@ module.exports = {
         path: `${__dirname}/src/content`,
       },
     },
+    // This plugin is set to disallow to prevent crawling of the site during preview
+    // mode
+    {
+      resolve: 'gatsby-plugin-robots-txt',
```

Comment:
I'm not sure how Gatsby handles this being in there "twice". We have this specified in the [gatsby theme](https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/gatsby-config.js#L17-L23) with a configurable value. Could you add these options as the `robots` key in the theme options? 

As you can see from the link, we have a default value set for the robots if you don't provide any overrides.

Reply:
will do.